### PR TITLE
Lachlanmackenzie/fix martins game update

### PR DESCRIPTION
**Pull request by lachlanin4**

### CODE REVIEW

File: quiz_game.py [hunk 1]
```diff
@@ -129,7 +134,27 @@ def ask_questions(self, questions):
                 print("Incorrect!")
         print("Contest completed!")
         print(f"You got: {total_points} points")
-    
+        print("Contest completed!")
+        print(f"You total is now: {total_points} points")
+
+
+
+        file = open('highscores.txt','a')
```
Comment (lachlanin4) [inline quiz_game.py:142]: What happens if the highscores.txt file exists but can't be written to?

File: quiz_game.py [hunk 2]
```diff
@@ -129,7 +134,27 @@ def ask_questions(self, questions):
...
+        file.write(name + " " + str(total_points) + "\n")
```
Comment (lachlanin4) [inline quiz_game.py:143]: If the file is opened as append will this just constantly add values for a given name even if they have improved or done the same on a previous session?

File: quiz_game.py (hunk 1 above)
Comment (lachlanin4) [reply quiz_game.py:140]: Something to think about for next time, but not at this iteration.

**Review by Sophiak113 — APPROVED** [review]

File: quiz_game.py (hunk 2 above)
Comment (lachlanin4) [reply quiz_game.py:141]: Something to think about for the future. Closing

**Comment by lachlanin4** [comment]
Merging since delta from last approval was removal of ""

**Pull request merged**